### ISSUE TITLE
Tabular figures not perfectly aligned

### ISSUE THREAD

**Issue by HazardCreative** [jpt/barlow#69]
Using as web font (WOFF & WOFF2) with `font-variant-numeric: tabular-nums`, I'm getting small but noticeable spacing variations between numerals. It looks great with static text, but is distracting when updated in real time:

![RH timer](https://user-images.githubusercontent.com/7169527/108649691-d6ec0400-748b-11eb-825c-aeec03f9c5a5.gif)

Thanks for this excellent typeface. I really appreciate the work that's gone in.

**Comment by jpt** [comment]
you're right, they broke.  I fixed them here, will update the issue when binaries are generated and published https://github.com/jpt/barlow/tree/1.5

**Comment by jpt** [comment]
ok, binaries are out here https://github.com/jpt/barlow/tree/1.5/fonts/ttf
testing and will pull in soon
